mcworldlib/logger.py

```diff
@@ -0,0 +1,6 @@
+import logging
+
+stdout_handler = logging.StreamHandler()
+stdout_handler.setLevel(logging.WARNING)
+log = logging.getLogger(__name__)
+log.addHandler(stdout_handler)
```

Comment:
I have a few issues with this approach:
- When you setup logging like this, moving `log = logging.getLogger(__name__)` to a dedicated module, `__name__` will always be `logger`, which kind of defeats the purpose. Each module should have their own `log` object named after themselves.
- "package-level" logging configuration _could_ be performed on `__init__.py`, but... a well-behaved library should not do _any_ logging configuration on its own, this is for the clients to do. Also, by default logging is _already_ set with a `StreamHandler` at `WARNING` level, so there's really no need for this module.

Reply:
That makes sense, I wasn't really thinking about that. I will go ahead and undo the logging changes.

Reply:
No problem! Once again, thanks for contributing!